source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
+                  <div xml:id="conditionColophone" type="div4">
+                     <head>Condition and Statement of Production</head>
+                  <p></p>
                   <p>
-                     <specList>
-                        <specDesc key="titlePage"/>
-                        <specDesc key="p"/>
-                        <specDesc key="table"/>
-                        <specDesc key="list"/>
-                        <specDesc key="quote"/>
-                        <specDesc key="lg"/>
+                        <specList>
+                        <specDesc key="condition"/>
+                        <specDesc key="colophone"/>
                      </specList>
-                  </p>
-                  <p>The following example shows the encoding of the title page of Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi>. Note the use of the <gi scheme="MEI">lb</gi> element to mark the line breaks present in the original.</p>
-                  <p>
-                     <figure>
-                        <head/>
-                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/text/text-sample371.txt" parse="text"/></egXML>
-                     </figure>
-                  </p>
-                  <p>The physical rendition of title page information is often of considerable importance. One approach to this requirement would be to use the <gi scheme="MEI">rend</gi> element, described in chapter <ptr target="#sharedTextRendition"/> to specify the rendition of each of the components of the title page. Another would be to employ a CSS stylesheet. Finally, a module customized for the description of typographic entities such as pages, lines, rules, etc., bearing special-purpose attributes to describe line-height, leading, degree of kerning, font, etc. could be employed.</p>
+                     </p>
+                  <p>Every physical part of a source is in a certain condition. The element <gi scheme="MEI">condition</gi> may contain plain text but also provides several elements that can be used for structuring the text if necessary (e.g. for linking) in order to describe the state or condition of a source.</p>
+                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see element definition “contained by”).</p>
+                     <p>The <gi scheme="MEI">condition</gi> element as a direct child of <gi scheme="MEI">physDesc</gi> should be used for describing the condition of a source in general. For describing special concerns like the condition of special parts of a source (e.g. the binding) the condition element can be used on the specific level. This approach promotes machine readability of the encoded metadata. In the end it depends on your encoding guidelines how detailed the description of the material condition is organized.</p>
```

Comment:
```suggestion
                     <p>For describing the condition of parts of a source (e.g. the binding) the <gi scheme="MEI">condition</gi> element can be used on the specific level. The level of detail you provide will depend on your specific needs for accurately recording the material condition.</p>
```

Reply:
The first sentence was redundant with other statements in this section.

An over-use of the word "special" was normalized. The word `condition` was wrapped in a `gi` element.

All markup promotes machine-readability, so this statement is not really necessary, particularly because this paragraph does not go into the specifics of what sort of machine readability it promotes. 

Final statement slightly re-worded to be a bit more formal (e.g., removing "in the end...") and makes it clear that it is about the needs of your project. (I think this was the essence of the original statement)